framework/configuration/entities-relations.rst

```diff
@@ -0,0 +1,402 @@
+Defines whether this entity will have a document tab or not. ``single_language`` is the only option that is currently in use.
+
+business-unit (*none*, single, manual_set, optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Defines how this entity handles Business-Units
+
+* *none*: Visible in all Business-Units. No relation to ``Business_unit`` will be created.
+
+* *single*: Only visible in the Business-Unit it was created in. A **n:1** relation to ``Business_unit`` will be created.
+
+* *manual_set*: Only visible in the Business-Unit that was selected. A **n:1** relation to ``Business_unit`` will be created.
+
+* *optional*: If a Business-Unit is set, only visible the selected Business-Unit. If nothing is set, every Business-Unit will see this entity. A **n:0..1** relation to ``Business_unit`` will be created.
+
+
+type (*standard*, lookup)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Differences between standard and lookup entities
+
+* There is no entity history for lookup entities
+
+* Relations to lookup entities will be displayed in simple Comboboxes in an Edit-Form. Relations to standard entities will be displayed as Remote-Fields.
+
+Field-Attributes
+----------------
+Field-Attributes are attributes of the ``<field>`` tag.
+
+.. code-block:: xml
+
+   <field name="label" type="string" localized="true">
+
+name (string)
+^^^^^^^^^^^^^
+The name of the entity-field. This name has to be unique for this entity-model. If this name is specified in more than one configuration, the system will merge them to a single entity-field.
+
+label (string)
+^^^^^^^^^^^^^^
+The label of this field. If not specified it will be a textresource whith the key following the rule ``entities.EntityName.fieldName``
+
+localized (true, *false*)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+If the field is localized, a field for each locale will be created. These fields will be called ``name_locale`` (e.g. ``label_de``).
+
+doc (string)
+^^^^^^^^^^^^
+Documentation of the entity, describing its meaning, purpose and important information to mention regarding the usage of the model. The rendering process of the entity model's documentation merges documentation into extending models and supports Markdown syntax. Therefore, the usage of Markdown is recommended to highlight important information or so tag specific terms as code elements. supports Markdown syntax.
+
+xss-filter (*true*, false)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+XSS-filter activates XssProtectionFieldValidator. Activated by default on any string-based field.
+
+unique (true, *false*)
+^^^^^^^^^^^^^^^^^^^^^^
+Adds a validator that checks a field value for uniqueness.
+
+type (string)
+^^^^^^^^^^^^^
+See `Field-Types`_.
+
+target (string)
+^^^^^^^^^^^^^^^
+The target of this field. For example the database-table-field. If this isn't specified, it will be the same as the name of the field.
+
+
+Field-Types
+-----------
+binary
+^^^^^^
+Field to store binaries. It will be stored as nullable ``VARCHAR(40)`` in a postgres db.
```

Comment:
Maybe a little more explanation? The VARCHAR(40) is only the reference to the binary.